frb/surveys/dlsurvey.py

```diff
@@ -14,8 +14,8 @@
 try:
     from dl import queryClient as qc, authClient as ac
     from dl.helpers.utils import convert
-except ImportError:
-    print("Warning:  You need to install dl")
+except:
+    print("Warning:  dl is not installed or will not properly connect")
```

Comment:
Replace "dl" in the print statement with "datalab-client" to prevent any confusion with this repo: https://github.com/mrdanbrooks/dl/blob/master/dl
If someone runs `pip install dl`, this is what they get.

Reply:
good suggestion.  done